src/sql/workbench/api/browser/mainThreadNotebookDocumentsAndEditors.ts

```diff
@@ -362,6 +362,17 @@ export class MainThreadNotebookDocumentsAndEditors extends Disposable implements
 		return Promise.resolve(this.doOpenEditor(resource, options));
 	}
 
+	$trySetTrusted(uri: UriComponents, isTrusted: boolean = true): Promise<boolean> {
```

Comment:
I'd get rid of the default value for isTrusted - this should always be explicitly specified